sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -15104,6 +15104,27 @@ public void testInGroupByOrderByLimitOutGroupByOrderByLimit()
         .run();
   }
 
+
+  @Test
+  public void testScanAndSortCanGetSchemaFromScanQuery()
+  {
+    skipVectorize();
+    cannotVectorize();
```

Comment:
Shouldn't have both of these: "skip" means skip the vectorization test completely, "cannot" means run the test and expect it to fail. Generally "skip" is best if the test fundamentally doesn't make sense under vectorization, and "cannot" is best if it makes sense in principle, but just can't vectorize today due to missing some implementation. That way, when someone adds the implementation in the future, the test here will automatically fail, and that future person will know to remove the `cannotVectorize()` line.

Perhaps we should change the test framework to make it an error to set both?

Reply:
yes - I'll change the testframework when I get to it ...  actually `cannotVectorize()` has a wierd issue: #15423